api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -59,10 +69,92 @@
     return ResponseEntity.ok(new ArrayList<>());
   }
 
+  private String createFirecloudUserAndBillingProject(User user) {
+    try {
+      fireCloudService.registerUser(user.getContactEmail(),
+          user.getGivenName(), user.getFamilyName());
+    } catch (ApiException e) {
+      log.log(Level.SEVERE, "Error registering user: {0}".format(e.getResponseBody()), e);
+      // We don't expect this to happen.
+      // TODO: figure out what happens if you register after already registering
+      throw new ServerErrorException("Error registering user", e);
```

Comment:
Is this a 500 Error? Is this the only error we expect to get back from FC here? (Its probably about time to start thinking about error codes, although I don't want to block this PR on that.)

Reply:
Yeah, let me start digging into the possible errors here. We definitely need better handling.

Reply:
OK. registerUser() here is idempotent... if you're already registered, it just updates the profile. I don't think there are other errors we need to handle here. Updated the comment to indicate this.